config/conference.ts

```diff
@@ -8,27 +8,26 @@ import {
   ImportantDate,
 } from './types'
 import venue from './venue'
-import { zonedTimeToUtc, utcToZonedTime } from 'date-fns-tz'
+import { zonedTimeToUtc } from 'date-fns-tz'
 import { add, sub, set, toDate } from 'date-fns'
 
 const name = 'DDD Perth'
 const tagLine = `${name} is an inclusive non-profit conference for the Perth software community`
-const tz = '+08:00'
 
 const hideDate = false
 const ticketPurchasingOptions = TicketPurchasingOptions.OnSale
 const staticDate = '2022-09-10T08:00'
-const date = utcToZonedTime(zonedTimeToUtc(staticDate, '+08:00'), tz)
+const date = zonedTimeToUtc(staticDate, '+08:00')
 const endDate = add(date, { hours: 12 })
 const currentInstance = date.getFullYear()
 const firstInstance = 2015
-const registrationOpenFrom = utcToZonedTime(zonedTimeToUtc('2022-05-16T08:00:00', '+08:00'), tz)
+const registrationOpenFrom = zonedTimeToUtc('2022-05-16T08:00:00', '+08:00')
```

Comment:
minor: could probably find/replace `+08:00` and replace with `Australia/Perth` or `conference.TimeZone` as appropriate, but no biggie

Reply:
@bplowry That's a good call. I also came across a variety of ways that dates & times were being formatted in the course of these recent changes, so might be worth refactoring the lot at some point.